test/binary_search_tree_test.rb

```diff
@@ -1,6 +1,5 @@
-require 'minitest/autorun'
-require 'minitest/pride'
-require './lib/binary_search_tree'
+require 'test_helper'
+
 
```

Comment:
All of your test files now have a single `require 'test_helper'`, that calls... the test helper!